.github/workflows/generated-linux-binary-manywheel-nightly.yml

```diff
@@ -25,6 +25,10 @@ env:
   BINARY_ENV_FILE: /tmp/env
   BUILD_ENVIRONMENT: linux-binary-manywheel
   BUILDER_ROOT: /builder
+  CU117_DEPENDENCIES: "nvidia-cuda-runtime-cu11==11.7.99;\
```

Comment:
Was this file manually edited ? This is generated file, should not be edited manually

Reply:
Also please keep the previous logic using PYTORCH_EXTRA_INSTALL_REQUIREMENTS